Source/MediaInfo/Export/Export_PBCore2.cpp

```diff
@@ -356,13 +356,15 @@ Ztring Export_PBCore2::Transform(MediaInfo_Internal &MI, version Version)
     else
         Format=__T("application/x-")+Ztring(MI.Get(Stream_General, 0, __T("Format"))).MakeLowerCase();
     Node_Main.Add_Child("instantiationDigital", Format);
-    
+
     //formatStandard
     Ztring formatStandard=MI.Get(Stream_General, 0, General_Format);
     if (!MI.Get(Stream_General, 0, General_Format_Commercial_IfAny).empty())
         formatStandard+=__T(" (")+MI.Get(Stream_General, 0, General_Format_Commercial_IfAny)+__T(")");
-    Node_Main.Add_Child("instantiationStandard", formatStandard);
-    
+    Node* Child=Node_Main.Add_Child("instantiationStandard", formatStandard);

```

Comment:
Again, i haven't tested this but adding these annotations makes sense as it somewhat mirrors the annotations in essenceTrackEncoding